skyportal/models/shift.py

```diff
@@ -179,3 +181,9 @@ class Shift(Base):
 ShiftUser.delete = CustomUserAccessControl(shiftuser_delete_access_logic)
 
 ShiftUser.create = ShiftUser.read
+
+Shift.shift_users_ids = column_property(
```

Comment:
Can we add some comment to explain what is happening here?

Reply:
Sure! Will do.